### ISSUE TITLE
Not able to parse checkboxes

### ISSUE THREAD

**Issue by Perlsowisdom** [alecdibble/logseq-journal-auto-copier#1]
The plugin seem to be unable to copy and delete a checkbox list. I'm not sure why I'm getting the errors I am.
I can only assume that somehow the brackets from the checkboxes is creating some sort of loop error causing the plugin to keep looking for a link. 
Below are my journal entries from the past two days.
[2022_12_14.md](https://github.com/alecdibble/logseq-journal-auto-copier/files/10239008/2022_12_14.md)
[2022_12_15.md](https://github.com/alecdibble/logseq-journal-auto-copier/files/10239009/2022_12_15.md)

**Comment by alecdibble** [comment]
@Perlsowisdom Thanks for reporting this! I will look into it this weekend and hopefully push up an update to resolve it.

**Comment by alecdibble** [comment]
@Perlsowisdom I was not able to replicate this behavior. When I copied your 12_14 entry into my test graph, it duplicated without errors. 

What platform are you running Logseq on? Are you on the latest version?

**Comment by Perlsowisdom** [comment]
I'm running Windows 11 version 22H2 on a Samsung Galaxy book 2. I just updated Logseq to v0.8.14.
So I think the Automatic Linker plugin, which couldn't understand manual checkboxes caused the excessive bracketing, and when they were attempted to be copied to a new journal entry, it caused them to loop linking only the brackets which caused the excess.

I disabled all my plugins and themes and that seemed to fix it, but it was still having issues parsing manual checkboxes with all my plugins enabled minus the automatic linker, so I need to figure out which plugin doesn't like yours 😅

Sorry for the false flag, and thanks for your help.
